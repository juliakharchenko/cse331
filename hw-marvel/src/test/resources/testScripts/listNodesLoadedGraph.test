# Tests that all the nodes from a file are printed

LoadGraph nba nbaPlayers.csv
ListNodes nba