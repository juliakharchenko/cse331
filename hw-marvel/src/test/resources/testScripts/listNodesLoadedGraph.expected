# Tests that all the nodes from a file are printed

loaded graph nba
nba contains: DWYANE-WADE KAREEM-ABDUL-JABBAR KOBE-BRYANT LEBRON-JAMES MAGIC-JOHNSON MICHAEL-JORDAN